ChangeLog

```diff
@@ -30,6 +30,7 @@ Changes
      through the symbol YOTTA_CFG_MBEDTLS_TARGET_CONFIG_FILE.
    * Added optimization for code space for X.509/OID based on configured
      features. (contributed by Aviv Palivoda)
+   * Renamed source file library/net.c to library/net_sockets.c.
```

Comment:
Add rationale - to avoid naming collision in projects which also have files with the common name 'net.c'.